shell/edit/cloudcredential.vue

```diff
@@ -80,20 +80,27 @@ export default {
     if ( this.value.provider ) {
       this.selectType(this.value.provider);
     }
+
+    this.selectCloudComponent();
   },
 
   data() {
     return {
       credCustomComponentValidation: false,
       nameRequiredValidation:        false,
       nodeDrivers:                   null,
-      kontainerDrivers:              null
+      kontainerDrivers:              null,
+      cloudComponent:                null
     };
   },
 
   watch: {
     'value._name'(newValue) {
       this.nameRequiredValidation = newValue?.length > 0;
+    },
+
+    driverName() {
```

Comment:
Driver name isn't always available by the time data/fetch are invoked so I needed to add a watcher.